### DISCUSSION TITLE
Can not install 3.0.0.beta.2

### DISCUSSION THREAD

**Discussion by lastmirage** [dynamoose/dynamoose#1393] [Q&A] [answered]
I want to use v3, so I install main branch with below command

npm install --save dynamoose/dynamoose#main

but it shows error like this

npm ERR! lib/Error.ts(1,27): error TS2307: Cannot find module 'dynamoose-utils' or its corresponding type declarations.
npm ERR! lib/utils/dynamoose/index.ts(6,38): error TS2307: Cannot find module 'dynamoose-utils' or its corresponding type declarations.

so I try to install 3.0.0.beta.1, and it is ok

what am I wrong? how can I fix it ?

**Comment by fishcharlie** [comment, marked as answer]
Duplicate of https://github.com/dynamoose/dynamoose/issues/1388.

Just install the beta from npm for now.
